src/Lookups/metadata.jl

```diff
@@ -96,6 +102,14 @@ end
 
 # Metadata utils
 
+"""
+Internal function.
```

Comment:
Does "Internal function" do something or its just a marker?

I'm not used to adding this much documentation, don't we just want this as a code comment?

Reply:
IMO this is very nice - when you see it in a stack trace, you can run `?` on it and immediately see what it does.  For people who are good at Julia, not 100% immersed in the DD world, and want to debug an error - this is priceless.

Reply:
I am wondering, whether the marker "Internal function" is even needed nowadays, because with the public keyword, and the changes in Base handling of the show of non-public functions this is rather handled by Base.